cmd/create/accountroles/cmd.go

```diff
@@ -215,36 +216,7 @@ func run(cmd *cobra.Command, argv []string) {
 	}
 
 	rolePath := args.rolePath
-	if interactive.Enabled() {
-		rolePath, err = interactive.GetString(interactive.Input{
-			Question: "Role Path",
-			Help:     cmd.Flags().Lookup("role-path").Usage,
-			Default:  rolePath,
-			Validators: []interactive.Validator{
-				aws.ARNPathValidator,
-			},
-		})
-		if err != nil {
-			r.Reporter.Errorf("Expected a valid path: %s", err)
-			os.Exit(1)
-		}
-	}
-
```

Comment:
I would honestly not remove these. How I've done it in the past is to only show the interactive mode for hidden flags if the user sets both the flag and interactive mode:
```go
if cmd.Flags().Changed("role-path") && interactive.Enabled() {
  ...
}
```
That way the code is still in place, but it only shows up if the user knows about the flags. It also means we can QE/test better to make sure the feature is ready. Then enabling it when it's ready becomes easier.

Reply:
done